wrappers/python/nntile/model/t5_lmhead.py

```diff
@@ -0,0 +1,168 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/model/t5_lmhead.py
+# T5 LMHead submodule of NNTile Python package
+#
+# @version 1.1.0
+# ruff: noqa: E501
+
+import torch
+from transformers.models.t5.modeling_t5 import (
+    T5ClassificationHead as T5ClassificationHeadTorch,
+    T5Config as T5ConfigTorch)
+
+import nntile.utils.constructors as nntc
+from nntile.layer.act import Act
+from nntile.layer.linear import Linear
+from nntile.model.base_model import BaseModel
+from nntile.model.t5_config import T5ConfigNNTile
+from nntile.tensor import TensorMoments, notrans, to_numpy
+
+
+class T5ClassificationHead(BaseModel):
+    next_tag: int
+
+    def __init__(
+        self, x: TensorMoments, config: T5ConfigNNTile, num_labels: int, next_tag: int
+    ):
+        """Head for sentence-level classification tasks."""
+        activations = [x]
+        layers = []
+        self.d_model = config.d_model
+        self.d_model_tile = config.d_model_tile
+        self.num_labels = num_labels
+        self.redux = config.redux
+
+        gemm_ndim = 1
+
+        # First linear layer (dense)
+        dense, next_tag = Linear.generate_simple(
+            x,
+            "R",
+            notrans,
+            gemm_ndim,
+            [self.d_model],
+            [self.d_model_tile],
+            next_tag,
+            redux=self.redux,
+            bias=True,
+        )
+
+        layers.append(dense)
+        activations.extend(dense.activations_output)
+
+        # Using gelutanh activation since tanh is not directly implemented
+        # This provides a non-linear activation similar to tanh
+        act_fn_layer, next_tag = Act.generate_simple(
+            activations[-1], "gelutanh", next_tag
+        )
+
+        layers.append(act_fn_layer)
+        activations.extend(act_fn_layer.activations_output)
+
+        # Output projection layer
+        out_proj, next_tag = Linear.generate_simple(
+            activations[-1],
+            "R",
+            notrans,
+            gemm_ndim,
+            [self.num_labels],
+            [self.num_labels],  # Using num_labels as tile size
```

Comment:
Generally, we shall use special value for tiling of this linear layer. But since it is the same for other models, we shall stick with it for now until we completely rework Python API.